src/tls/mod.rs

```diff
@@ -19,26 +16,27 @@ mod settings;
 #[cfg(feature = "sources-tls")]
 pub(crate) use incoming::{MaybeTlsIncomingStream, MaybeTlsListener};
 pub(crate) use maybe_tls::MaybeTls;
-pub(crate) use outgoing::MaybeTlsConnector;
 pub use settings::{MaybeTlsSettings, TlsConfig, TlsOptions, TlsSettings};
 
+// TODO: Rename to TlsResult?
 pub type Result<T> = std::result::Result<T, TlsError>;
 
 pub type MaybeTlsStream<S> = MaybeTls<S, SslStream<S>>;
 
+// TODO: check what not used and remove
```

Comment:
Are these still outstanding?

Reply:
Yes :( need check everything, make slight changes and remove draft pr mark.